src/sysfiles.c

```diff
@@ -3109,7 +3153,7 @@ UInt SyExecuteProcess (
       func2 = &NullSignalHandler;
 
     /* clone the process                                                   */
-    pid = vfork();
```

Comment:
I think the use of `vfork` here is/was meant as an optimization that preserves memory and is also easier to implement on, say, Windows, than full `fork`. This PR silently changes it to `fork`, a deoptimization. Do we really want that? Perhaps a `SyVFork` is needed, too?

Reply:
The spec for `vfork` says the only functions you are allowed to call after a `vfork` is `_exit` or `exec*`. We are calling `chdir`, `open`, `fcntl` and `SyBufFileno` (which is one of our functions, but I believe even that isn't acceptable). I can't find any spec that says that's acceptable, even on Linux. However, I shouldn't have bundled this into the same PR, as if we want to keep vfork a SyVfork would be easy to add.

Reply:
Actually, cygwin's vfork just calls fork it turns out ( https://github.com/wallento/newlib-cygwin/blob/dbe905c140d0cd347c731f63b153dca53857bdc9/newlib/libc/sys/linux/process.c#L32 ), although I think linux vfork actually does "proper" vfork.